packages/server/src/seeds/searchparameters.ts

```diff
@@ -4,19 +4,32 @@ import { getDatabasePool } from '../database';
 import { Repository, getSystemRepo } from '../fhir/repo';
 import { globalLogger } from '../logger';
 import { r4ProjectId } from '../seed';
+import { RebuildOptions, buildRebuildOptions } from './common';
 
 /**
  * Creates all SearchParameter resources.
+ * @param options - Optional options for how rebuild should be done.
  */
-export async function rebuildR4SearchParameters(): Promise<void> {
+export async function rebuildR4SearchParameters(options?: Partial<RebuildOptions>): Promise<void> {
+  const finalOptions = buildRebuildOptions(options);
```

Comment:
A long time ago, I was scolded in a code review for using the variable name "finalOption", because it sounded too close to "final solution", which has some horrific connotations. 🤷

Reply:
Open to suggestions to indicate the finality of the options without using this verbiage

Reply:
Went with `rebuildOptions`